src/main/topLevelPages/myDashboard/newMemberships/NewMembershipsBlock.tsx

```diff
@@ -111,17 +111,17 @@ const NewMembershipsBlock = ({
     [data?.me.applications]
   );
 
-  const pendingApplications = useMemo(
+  const pendingApplications = applications.filter(invitation => !RECENT_MEMBERSHIP_STATES.includes(invitation.state));
+
+  const recentPendingApplications = useMemo(
     () =>
-      sortBy(
-        applications.filter(invitation => !RECENT_MEMBERSHIP_STATES.includes(invitation.state)),
-        ({ createdDate }) => createdDate
-      )
+      sortBy(pendingApplications, ({ createdDate }) => createdDate)
         .reverse()
         .slice(0, Math.max(0, PENDING_MEMBERSHIPS_MAX_ITEMS - pendingInvitations.length)),
     [applications]
   );
-  const pendingMembershipsCount = pendingInvitations.length + pendingApplications.length;
+
+  const pendingMembershipsCount = pendingInvitations.length + recentPendingApplications.length;
```

Comment:
shouldn't this be `pendingApplications`? I see it is keeping the same functionality, but I think it is wrong.. if there is a counter it should show the total, not the sliced. Haven't tested it